040-SDK/101-get.mdx

```diff
@@ -799,7 +799,7 @@ You can continue like this until `"more"` is returned as `false`.
 
 ### Offset-based Pagination
 
-The offset-page pagination is limited to querying up to 1000 records and is recommended only for usecases where you don't expect the number of records to grow beyond that. In most cases, you should use the cursor-based pagination.
+The offset-page pagination limit is 50,000 records (about 250 pages). In most cases, you should use the cursor-based pagination.
```

Comment:
I think we don't need the 250 pages limit, that depends on the page size, but what we actually limit is the page offset, which is limited to 49K.

```suggestion
The offset-page pagination limit is 50,000 records. In most cases, you should use the cursor-based pagination.
```